rest-api/tools/import_organizations.py

```diff
@@ -0,0 +1,156 @@
+"""Imports awardees / HPOs, organizations, and sites into the database.
+
+Usage:
+  tools/import_organizations.sh [--account <USER>@pmi-ops.org --project <PROJECT>] \
+      --awardee_file <AWARDEE CSV> --organization_file <ORGANIZATION CSV> \
+      --site_file <SITE CSV> [--dry_run]
+
+  The CSV files originate from the Google Sheets found here:
+
+  https://docs.google.com/spreadsheets/d/1CcIGRV0Bd6BIz7PeuvrV6QDGDRQkp83CJUkWAl-fG58/edit
+"""
+
+import logging
+
+from dateutil.parser import parse
+from tools.csv_importer import CsvImporter
+from dao.hpo_dao import HPODao
+from dao.organization_dao import OrganizationDao
+from dao.site_dao import SiteDao
+from model.hpo import HPO
+from model.organization import Organization
+from model.site import Site
+from model.site_enums import SiteStatus
+from participant_enums import OrganizationType
+from main_util import get_parser, configure_logging
+
+class HPOImporter(CsvImporter):
+
+  def __init__(self):
+    super(HPOImporter, self).__init__('awardee', HPODao(), 'hpoId', 'name',
+                                      ['Awardee ID', 'Name', 'Type'])
```

Comment:
nit: constants? seems easy to fatfinger unstructured strings like this (awardee ID in particular)

Reply:
Done.